src/app/_utils/code-example.ts

```diff
@@ -5,7 +5,7 @@
  */
 
 import {Component, Input, ViewChild} from "@angular/core";
-import {CodeHighlight} from "../../clarity-angular/code/code-highlight";
+import {SyntaxHighlight} from "../../clarity-angular/code/syntax-highlight/syntax-highlight";
```

Comment:
Is this a breaking change? If yes, shouldn't this go in 0.10.0? This will definitely break the website because we Import `CodeHighlight` in the website.